typescript/packages/common-runner/src/cell-map.ts

```diff
@@ -160,4 +162,24 @@ class CleanableMap<T extends object> {
   }
 }
 
-const entityIdToDocMap = new CleanableMap<DocImpl<any>>();
+export function getDocByEntityId<T = any>(
```

Comment:
this seems fragile/scary - in that the ID isn't a URN that contains the space as part of its name...

Reply:
meaning the entityid has been confusing (with '/': uint vs '/': string vs raw string) and now an entityId isn't necessarily unique in the instantiated system